### DISCUSSION TITLE
How to filter by foreign table row count?

### DISCUSSION THREAD

**Discussion by laubonghaudoi** [supabase/supabase#15746] [Questions]
According to the [official doc](https://supabase.com/docs/reference/javascript/using-filters), we can filter the roles by the foreign table:
```typescript
const { data, error } = await supabase
  .from('countries')
  .select(`
    name,
    cities!inner (
      name
    )
  `)
  .eq('cities.name', 'Bali')
```
And we can also get the row count of foreign tables:
```typescript
const { data, error } = await supabase
  .from('countries')
  .select(`*, cities(count)`)
```
However, how can I filter the rows by foreign table row count, like this:
```typescript
const { data, error } = await supabase
  .from('countries')
  .select(`*, cities(count)`)
  .gt(`cities.count`, 2)
```
Is this possible in Supabase? I just tried that and it gave me this error:
```
Server Error
Error: {"code":"42803","details":null,"hint":null,"message":"aggregate functions are not allowed in WHERE"}
```

**Comment by steve-chavez** [comment]
> Is this possible in Supabase? I just tried that and it gave me this error:

Not yet. Please subscribe to https://github.com/PostgREST/postgrest/issues/2730.
